src/test/java/io/stargate/sgv2/jsonapi/api/v1/CollectionResourceIntegrationTest.java

```diff
@@ -82,6 +82,35 @@ public void unknownCommand() {
               startsWith("No \"unknownCommand\" command found as \"CollectionCommand\""));
     }
 
+    @Test
+    public void unknownCommandField() {
+      String json =
+          """
+              {
+                "findOne": {
+                    "unknown": "value"
+                }
+              }
+              """;
+
+      given()
+          .headers(getHeaders())
+          .contentType(ContentType.JSON)
+          .body(json)
+          .when()
+          .post(CollectionResource.BASE_PATH, namespaceName, collectionName)
+          .then()
+          .statusCode(200)
+          .body("errors", hasSize(1))
+          .body("errors[0].errorCode", is("INVALID_REQUEST_UNKNOWN_FIELD"))
+          .body("errors[0].exceptionClass", is("JsonApiException"))
+          .body("errors[0].message", startsWith("Request invalid, unrecognized JSON field"))
+          .body("errors[0].message", containsString("\"unknown\" not one of known fields"))
```

Comment:
Just checking but I presume this error message will also contain `at 'findOne'`? Having the full path in the error message similar to #1208 would be ideal.

Reply:
Unfortunately it depends on location (in JSON doc) of mismatch. #1208 is special-casing matching and same could be done elsewhere but general handling added in this case is invoked if no special handler is matched.
I suspect `findOne` is not included as it is type identifier used to match specific command subtype. Other than that path is indeed indicated.

We can add special handling for commonly encountered cases but effort to do it for everything is prohibitive

I'll see what is the exact message being given just to make sure.

Reply:
Ok, so, "findOne" is kind of indicated: 

```
Request invalid, unrecognized JSON field: \"unknown\" 
....
 at 'io.stargate.sgv2.jsonapi.api.model.command.impl.FindOneCommand["unknown"]'
```

which is not ideal (as it refers to Java class names), but is the path we can get out of exception.